tests/kafkatest/services/security/security_config.py

```diff
@@ -23,20 +24,19 @@
 
 class SslStores(object):
     def __init__(self):
-        self.ca_crt_path = "/tmp/test.ca.crt"
-        self.ca_jks_path = "/tmp/test.ca.jks"
+        self.ca_and_truststore_dir = mkdtemp(dir="/tmp")
+        self.ca_crt_path = os.path.join(self.ca_and_truststore_dir, "test.ca.crt")
+        self.ca_jks_path = os.path.join(self.ca_and_truststore_dir, "test.ca.jks")
         self.ca_passwd = "test-ca-passwd"
 
-        self.truststore_path = "/tmp/test.truststore.jks"
+        self.truststore_path = os.path.join(self.ca_and_truststore_dir, "test.truststore.jks")
         self.truststore_passwd = "test-ts-passwd"
         self.keystore_passwd = "test-ks-passwd"
         self.key_passwd = "test-key-passwd"
         # Allow upto one hour of clock skew between host and VMs
         self.startdate = "-1H"
-
-        for file in [self.ca_crt_path, self.ca_jks_path, self.truststore_path]:
-            if os.path.exists(file):
-                os.remove(file)
+        # Register rmtree to run on exit
+        atexit.register(rmtree, self.ca_and_truststore_dir)
 
```

Comment:
Is`atexit` is the right approach here? The registered function will only run when the entire ducktape process finishes (i.e. *after* all ~215 tests finish). Use of the `tempfile.mkdtemp` will still avoid path collisions from concurrent processes, so maybe it's good enough.

Another possibility that at least results in immediate cleanup:
put directory removal into `clean_node` (but check the containing directory is present before removing to avoid errors)